### DISCUSSION TITLE
A typo in pybullet.c

### DISCUSSION THREAD

**Discussion by liusida** [bulletphysics/bullet3#3721] [General]
https://github.com/bulletphysics/bullet3/blob/master/examples/pybullet/pybullet.c#L8056

I think here, `collisionShapeOrientationA` should be `collisionShapeOrientationAObj`, though this is not mentioned in the documentation, I think it is still worth to correct it.

**Comment by erwincoumans** [comment]
Good find, that is a copy/paste bug. The documentation still refers to collisionShapeOrientationA anyway, but the internal implementation should be collisionShapeOrientationAObj indeed.

**Comment by erwincoumans** [comment]
Fixed here: https://github.com/bulletphysics/bullet3/pull/2986
